compose/cli/main.py

```diff
@@ -694,6 +694,7 @@ def ps(self, options):
             -q, --quiet          Only display IDs
             --services           Display services
             --filter KEY=VAL     Filter services by a property
+            -a, --all            Show all stopped containers
```

Comment:
Can we add `(including those created by the run command)` to the description to make this clearer?